packages/drivers/web-utils/src/utils.ts

```diff
@@ -123,16 +123,9 @@ export function createMarkedViewHierarchy() {
         .filter(
           (attr) =>
             attr.name.match(/^aria-/i) ||
-            [
-              "href",
-              "target",
-              "src",
-              "alt",
-              "type",
-              "name",
-              "value",
-              "role",
-            ].includes(attr.name.toLowerCase()),
+            ["target", "alt", "type", "name", "value", "role"].includes(
```

Comment:
feels like these attributes wants to be matchers as well

Reply:
same with the aria attributes, otherwise we don't count them when we score the closest element with our matcher

Reply:
not under scope of this change, so I opened a task for it: https://github.com/wix-incubator/pilot/issues/113